### PR TITLE
style-guide: move language specific rules out of style guide

### PR DESCRIPTION
**Pull request by blueskyson**
As #8700 introduced French specific rules, we have two languages' rules. It's time to have an independent document for language specific rules to make style-guide.md less complex and more focusing on tldr's syntax.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -115,7 +115,9 @@ Some examples of valid locale tags:
  - Chinese (Singapore): `zh_SG`.
  - Portuguese (Brazil): `pt_BR`.
 
-A list of translated templates for alias pages can be found in [here](contributing-guides/translation-templates/alias-pages.md).
+A list of translated templates for alias pages can be found [here](contributing-guides/translation-templates/alias-pages.md).
+
+Specific rules of Chinese and French can be found [here](contributing-guides/language-specific-rules.md).
```
Comment (kbdharun) [inline CONTRIBUTING.md:120]:
```suggestion
Language specific rules for pages can be found [here](contributing-guides/language-specific-rules.md).
```
I think having this generalised would be good. As many more languages will be added to this list. For example, @reinhart1010 is working on rewriting all Indonesian pages in #8576.

**Comment by mcombeau** [comment]
Great idea, moving the language rules to a separate file! One thing I would suggest is also changing the line under the "Imperative mood" section which currently says :

>This also applies to all translations by default, unless otherwise specified in the language-specific section below.

Including another link to the new language rule file in this line would be useful. (Sorry, I would've suggested this small change in a review directly, but GiHub seems to think that section is too far away from your changes :sweat_smile:)

**Comment by blueskyson** [comment]
@mcombeau Thanks for your reviewing. I didn't noticed that line, and now I fix it.

**Comment by kbdharun** [comment]
@blueskyson I made some conflicting changes to the style guide in #8858. I think we can merge this PR after #8858 is merged.
Once the style guide is updated, You can sync this PR's branch with `main`.

**Comment by kbdharun** [comment]
> @blueskyson I made some conflicting changes to the style guide in #8858. I think we can merge this PR after #8858 is merged. Once the style guide is updated, You can sync this PR's branch with `main`.

Now that #8858 is merged we can work with this PR. Sync your branch with the main. Thanks.

**Pull request closed without merging**